AUTHORS

```diff
@@ -2316,5 +2315,374 @@ Ben Newman <ben@meteor.com>
+Elian Gutierrez <eliangu3600103@gmail.com>
+Jose Bucio <jbucio@shoprunner.com>
+Thomas GENTILHOMME <gentilhomme.thomas@gmail.com>
+Romain Lanz <romain.lanz@slynova.ch>
+Andre Jodat-Danbrani <andre.jodat-danbrani@xe.com>
+Nolan Rigo <nolan@rigo.pro>
+Jonathan Cardoso Machado <me@jonathancardoso.com>
+Raoof <raoofhb@gmail.com>
+Loic <loic.veteau@outlook.fr>
+Denis McDonald <denis.l.mcdonald@gmail.com>
+Arvind Pandey <arvind3157@gmail.com>
+Jagannath Bhat <jagannathbhat1998@gmail.com>
+Geoffrey Booth <webmaster@geoffreybooth.com>
+mritunjaygoutam12 <mritunjaygoutam2204@gmail.com>
+Esteban Sotillo <westixy@gmail.com>
+Jerome Covington <jeromecovington@gmail.com>
+Rob Reynolds <ferventcoder@gmail.com>
+warnerp18 <warnerp18@gmail.com>
+chux0519 <chuxdesign@hotmail.com>
+Tadhg Creedon <tadhg.creedon@rangle.io>
+Petar Dodev <petar.dodev@gmail.com>
+mzucker <michaeljzucker@gmail.com>
+Morgan Roderick <morgan@roderick.dk>
+Remy Parzinski <rs.parzinski@gmail.com>
+Roland Broekema <roland.broekema@qnh.nl>
+Florin-Daniel BÎLBÎE <fbilbie@bitdefender.com>
+Robin Drexler <robin.drexler@xing.com>
+ZauberNerd <zaubernerd@zaubernerd.de>
+G. Carcaci <giuseppecarcaci@gmail.com>
+Jackson Chui <14085209+haiXchuus@users.noreply.github.com>
+Marc Posth <marc.posth@shopify.com>
+Simona Cotin <simona.cotin@gmail.com>
+Mathieu Pavageau <mth.pvg@gmail.com>
+Clement <clement.sch@free.fr>
+razvanbh <razvy.rotaru@gmail.com>
+reineke-fox <nils@reineke.io>
+Mark Arranz <mark.arranz@gmail.com>
+alyssaq <alyssaquek@gmail.com>
+Manish Poddar <manish.poddar@epoch.com>
+szabolcsit <szabolcsit@hu.ibm.com>
+Kevin Seidel <kevin.seidel@karrieretutor.de>
+Jonah Polack <jonah.polack@gmail.com>
+Paul Isache <paul@isache.com>
+Artur Daschevici <artur.daschevici@nearform.com>
+Grant Carthew <grant@carthew.net>
+Alex Seifert <alex@alexseifert.com>
+Amer Alimanović <x@amer8.com>
+Nikita Malyschkin <nikita.malyschkin@gmail.com>
+ivan.filenko <filenko@uchi.ru>
+Marcus Scott <mmmrscott@gmail.com>
+Berry de Witte <bdwitte@wehkamp.nl>
+saurabhSiddhu <sourabh.siddhu@gmail.com>
+Marie Terrier <marie.terrier1@gmail.com>
+John Mc Quillan <john.mcq1llan@gmail.com>
+Ulises Santana Suárez <ulisesantana@gmail.com>
+Yehiyam Livneh <yehiyam@gmail.com>
+Aivo Paas <aivo.paas@pipedrive.com>
+James Herrington <james@getimpala.com>
+Natalie Cluer <natalie@getimpala.com>
+Dmitry Igrishin <dmitigr@gmail.com>
+Martin Kask <martin.kask@pipedrive.com>
+Alessandro Gatti <ale.gatti96@gmail.com>
+Vladyslav Kopylash <vladik.kopilash@gmail.com>
+Osmond van Hemert <osmond@vanhemert.nu>
+msmichellegar <msmichellegar@gmail.com>
+Robert Pamely <info@robertpamely.com>
+Lauri Piisang <lauri.piisang@pipedrive.com>
+kiyomizumia <melissa@stroly.jp>
+Paul Hodgson <prhodgson1@gmail.com>
+Emanuel Kluge <kraemerseele@gmx.de>
+Andrey Melikhov <amel.truelife@gmail.com>
+Philipp Dunkel <pip@pipobscure.com>
+Steven Gabarro <bewchabbacc@gmail.com>
+Anatoli Papirovski <anatoli.papirovski@postmates.com>
```

Comment:
I think this is @apapirovski, already listed under https://github.com/nodejs/node/blob/0aeb028d1e72ed9465a069cd5710c2c578bdb5cd/AUTHORS#L1655

Reply:
Thanks, done :+1: